docs/api/navigators/DrawerNavigator.md

```diff
@@ -88,7 +88,7 @@ The route configs object is a mapping from route name to a route config, which t
 
 - `drawerWidth` - Width of the drawer
 - `drawerPosition` - Options are `left` or `right`. Default is `left` position.
-- `contentComponent` - Component to use to render the navigation items. Receives the `navigation` prop for the drawer. Defaults to `DrawerView.Items`.
+- `contentComponent` - Component to use to render a container with the navigation items. Receives the `navigation` prop for the drawer. Defaults to `DrawerView.Items`.
```

Comment:
Maybe this is better:

`Component to to render the content of the drawer, e.g. - navigation items.`

A container usually means wrapper or redux containers. You're rendering the content and not a container. I think  the example clears it up.